gravitee-am-gateway/gravitee-am-gateway-services/gravitee-am-gateway-services-sync/src/main/java/io/gravitee/am/gateway/services/sync/SyncManager.java

```diff
@@ -140,6 +146,11 @@ public void afterPropertiesSet() throws Exception {
     }
 
     public void refresh() {
+        if (reactorNotStarted()) {
```

Comment:
please write a test for this case 

It should be Lifecycle.state mocked for other tests too